src/main/java/games/strategy/triplea/util/TuvUtils.java

```diff
@@ -129,6 +136,20 @@ private TuvUtils() {}
     return costs;
   }
 
+  private static int getTotalTuv(UnitType unitType, IntegerMap<UnitType> costs, Set<UnitType> alreadyAdded) {
```

Comment:
Nit: Running Clean Up here will mark the method parameters `final`.

Reply:
Good catch. Updated.